dbms/src/Flash/Coprocessor/ArrowColCodec.cpp

```diff
@@ -347,9 +347,8 @@ void flashEnumColToArrowCol(
                 enum_value,
                 enum_value_size);
 
-        TiDBEnum ti_enum(
-            enum_value,
-            enum_type->getNameForValue(static_cast<const DataTypeEnum16::FieldType>(enum_value)));
+        const auto & enum_name = enum_type->getNameForValue(static_cast<const DataTypeEnum16::FieldType>(enum_value));
+        TiDBEnum ti_enum(enum_value, enum_name);
```

Comment:
I think it's better to change the `TiDBEnum`'s `name` type to `StringRef`, as the copy of `StringRef` is very small.

Reply:
Sounds good, change it.